### ISSUE TITLE
My accuracy is 75.6%

### ISSUE THREAD

**Issue by zswzifir** [yjxiong/anet2016-cuhk#27]
Hi, yuanjun,
Thanks for your code and models. 
I run your code and models on activitynet1.3, but the top1 accuracy is just 75.6% (RGB stream), I have no idea why.
I fit up your code with the video path.
What maybe the reasons, can you give me some ideas?
Thanks very much.

**Comment by yjxiong** [comment]
I don’t think we released the evaluation code on this dataset so I don’t know how to answer this question. Some information on how you did it would help here.

**Comment by zswzifir** [comment]
Thanks for reply.
 I just evaluate the top1 accuray roughly, and do not consider the multi action instance with different labels. The following is my code:
`
db = ANetDB.get_db("1.3")
lb_list = db.get_ordered_label_list()
total = 0
cro = 0
cls = ActionClassifier(models, dev_id=2)
vid_dir = './Activitynet/anet_1.3_video_val'
 json_file = './ActivityNet_v1.3/activity_net.v1-3.min.json'
list_file = 'data/vid.txt'
with open(json_file, 'r') as fid:
     info = json.load(fid)['database']
with open(list_file, 'r') as fid:
     vid_list = fid.readlines()
print('Appying videos ({})...'.format(list_file))
for i, vid in  enumerate(vid_list):
        total += 1
        pth = os.path.join(vid_dir, vid[:-1])

        key = vid.split('.')[0]
        key = vid[7:18]
        key_info = info[key]
        anns = key_info['annotations']

        rst = cls.classify(pth)
        scores = rst[0]
        idx = np.argsort(scores)[::-1]

        k = idx[0]
        pre_name = lb_list[k]
        print('ID:{}, category:'.format(key))

        for an in anns:
            print(an['label'], ',', )
            if pre_name == an['label']:
                cro += 1
                break
        print('\t* cur accuray:{}({}/{})'.format(cro / total, cro, total))
        print('----------------Classification Results----------------------')
        for i in range(10):
            k = idx[i]
            print('\t', lb_list[k], scores[k])

`
